.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: End-to-end tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./utility-emissions-channel
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12.15.0'
+      - name: Download Fabric binaries and Docker compose
+        run: |
+          sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
Docker-compose should be pre-installed on `ubuntu-latest`. Could you try and verify this, please?
Or do you need a specific version?

Reply:
Docker compose wasn't installed by default, so I had to install it with the script.

Reply:
okay. Could you take a look into this, please? https://github.com/peter-evans/docker-compose-actions-workflow
It looks like `docker-compose` is pre-installed.
However, if it is not pre-installed you could check out this GHA: https://github.com/KengoTODA/actions-setup-docker-compose

Reply:
Hello, @udosson. I had a look at the pre-installed tools in the [ubuntu-latest image](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) and Docker-compose does seem to be installed. I have removed the line. Also, based on what @Zzocker said, I've removed the `-it` flag from all the Docker commands. All the Docker containers appear to be starting normally now.